### PR TITLE
[es] Fix link

### PR DESCRIPTION
**Pull request by Arhell**
en PR https://github.com/kubernetes/website/pull/45849/files

### CODE REVIEW

File: content/es/docs/concepts/services-networking/ingress.md [hunk 1]
```diff
@@ -737,7 +737,7 @@ recurso Ingress:
 * Usa
   un [Service.Type=LoadBalancer](es/docs/concepts/services-networking/service/#loadbalancer)
 * Usa
-  un [Service.Type=NodePort](es/docs/concepts/services-networking/service/#nodeport)
+  un [Service.Type=NodePort](es/docs/concepts/services-networking/service/#type-nodeport)
```
Comment (ramrodo) [inline content/es/docs/concepts/services-networking/ingress.md:740]: The correct reference is different in the Spanish `service.md` file, you can see it [here](https://github.com/kubernetes/website/blob/main/content/es/docs/concepts/services-networking/service.md?plain=1#L406).

```suggestion
  un [Service.Type=NodePort](es/docs/concepts/services-networking/service/#tipo-nodeport)
```

**Review by ramrodo — CHANGES_REQUESTED** [review]
Thanks, @Arhell !

I made a suggestion related to the Spanish reference.

File: content/es/docs/concepts/services-networking/ingress.md (hunk 1 above)
Comment (Arhell) [reply content/es/docs/concepts/services-networking/ingress.md]: Done, thanks

**Comment by ramrodo** [comment]
Thanks!
/lgtm

**Pull request merged**